app/src/main/java/com/example/timeman/MainActivity.kt

```diff
@@ -45,10 +37,11 @@ class MainActivity : AppCompatActivity() {
     }
 
     private fun startStopTimer() {
-        if (timerStarted)
+        if (timerStarted) {
             stopTimer()
-        else
+        } else {
             startTimer()
+        }
```

Comment:
fixed syntax of if else statement - https://kotlinlang.org/docs/coding-conventions.html

Reply:
Thank you